src/main/java/net/imagej/ops/create/CreateNamespace.java

```diff
@@ -543,6 +543,33 @@ public Object kernelLog(final Object... args) {
 		return result;
 	}
 
+	// -- kernelSobel --
+
+	@OpMethod(op = net.imagej.ops.create.kernelSobel.CreateKernelSobelSeparated.class)
+	public <T extends ComplexType<T>> Img<T> kernelSobelSeparated() {
+		@SuppressWarnings("unchecked")
+		final Img<T> result = (Img<T>) ops().run(
+			net.imagej.ops.create.kernelSobel.CreateKernelSobelSeparated.class);
+		return result;
+	}
+
+	@OpMethod(op = net.imagej.ops.create.kernelSobel.CreateKernelSobelSeparated.class)
+	public <T extends ComplexType<T>> Img<T> kernelSobelSeparated(final Type<T> outType) {
```

Comment:
Yes, always use `T`, not `Type<T>`. I actually just fixed all of the known offenders with 9316184ca99e9ca34b05e414b3a80fb51e352340. Let's try to keep things clean!